data/sles4sap/agama/sles_sap_default_staging_baremetal.jsonnet

```diff
@@ -0,0 +1,77 @@
+local repo = '{{INCIDENT_REPO}}';
+local urls = std.split(repo, ",");
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE_SLES4SAP}}',
+  },
+  bootloader: {
+    stopOnBootMenu: true
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+    sshPublicKey: 'enable ssh',
+  },
+  software: {
+    packages: [],
+    extraRepositories:
+      if repo != "" then
+        [
+          {
+            alias: "TEST_" + std.toString(i),
+            url: urls[i],
+            allowUnsigned: true
+          }
+          for i in std.range(0, std.length(urls) -1)
+        ]
+      else
+        [],
+    onlyRequired: false
+  },
+  questions: {
+    policy: 'auto',
+    answers: [
+      {
+        answer: 'Trust',
+        class: 'software.import_gpg'
+      }
+    ]
+  },
+  storage: {
+    drives: [
+      {
+        search: '/dev/sda',
```

Comment:
I am not sure if it fit every bare metal server, for example some BM has no hdd but NVMe luns instead. I would suggest to remove this line and the system will be installed on to the first "device" recognized by udev